docs/csharp/programming-guide/types/toc.yml

```diff
@@ -5,14 +5,13 @@
     href: casting-and-type-conversions.md
```

Comment:
same here, first node needs to be expanded in order to be a parent node

Reply:
same answer...